greengrass_ipc/source/platform/PosixConnectionConfig.cpp

```diff
@@ -0,0 +1,33 @@
+#include <aws/greengrass/GreengrassCoreIpcClient.h>
```

Comment:
Keep this version of the file, remove the Window's version and suppress the warning.

Likely we will want to have an aws_getenv() function added to handle whatever the best behavior per platform should be.

Reply:
Okay, I'll say that this is something we can do later on as introducing this may require more work. For now, we can suppress the warning in MSVC:
```
#if defined(_MSC_VER)
    #pragma warning(disable: 4996)
#endif